lib/3scale/backend/stats/cleaner.rb

```diff
@@ -108,6 +98,25 @@ def delete!(redis_conns, log_deleted_keys: nil)
 
           private
 
+          def handle_redis_exception(exception, redis_conn)
+            # If it's a connection error, simply log so we can continue with
+            # other shards. If it's another kind of error, it could be caused by
+            # a bug, so better re-raise.
+
+            is_connection_err = REDIS_CONN_ERRORS.any? do |conn_error|
+              exception.is_a?(conn_error)
```

Comment:
`is_a?` is not what `rescue` uses to discriminate exceptions. Instead it uses `===` (_threequals_).

Reply:
No need to fix because https://github.com/3scale/apisonator/pull/250#discussion_r564722826